app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -46,6 +50,7 @@
 .gem-c-pagination__link-text {
   @include govuk-font(19, $weight: bold);
   margin-left: govuk-spacing(2);
+  color: $govuk-text-colour;
```

Comment:
This class is inside `link-title`, where the colour is already set, does it need to be set twice?

Reply:
removed the unneeded second setting of the color